test/e2e/Makefile

```diff
@@ -0,0 +1,5 @@
+SHELL = /bin/bash
+
+run:
+	go run github.com/onsi/ginkgo/v2/ginkgo run --tags E2Etests --junit-report ./report-$(shell date --iso-8601).xml ./
```

Comment:
Here I would rather use a timestamp with at least minute precision, so something like `date -Iminute` or sheer unix epoch timestamp like `date +'%s'`, so that someone rerunning tests locally don't accidently overwrite previous test results.

```
$ date -Iminute
2025-03-20T17:07+01:00
```

```
$ date +'%s'
1742486970
```